common/logging_cli.go

```diff
@@ -0,0 +1,127 @@
+package common
+
+import (
+	"fmt"
+	"io"
+	"log/slog"
+	"os"
+
+	"github.com/Layr-Labs/eigenda/common"
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	"github.com/urfave/cli/v2"
+)
+
+/*
+	TODO: https://github.com/Layr-Labs/eigenda-proxy/issues/268
+
+	This CLI logic is already defined in the eigenda monorepo:
+	 https://github.com/Layr-Labs/eigenda/blob/0d293cc031987c43f653535732c6e1f1fa65a0b2/common/logger_config.go
+
+	This regression is due to the fact the proxy leverage urfave/cli/v2 whereas
+	core eigenda predominantly uses urfave/cli (i.e, v1).
+
+*/
+
+const (
+	PathFlagName   = "path"
+	LevelFlagName  = "level"
+	FormatFlagName = "format"
+	// deprecated
+	PidFlagName   = "pid"
+	ColorFlagName = "color"
+)
+
+type LogFormat string
+
+const (
+	JSONLogFormat LogFormat = "json"
+	TextLogFormat LogFormat = "text"
+)
+
+type LoggerConfig struct {
+	Format       LogFormat
+	OutputWriter io.Writer
+	HandlerOpts  logging.SLoggerOptions
+}
+
+func LoggerCLIFlags(envPrefix string, flagPrefix string) []cli.Flag {
+	category := "logging"
+
+	return []cli.Flag{
+		&cli.StringFlag{
+			Name:     common.PrefixFlag(flagPrefix, LevelFlagName),
+			Category: category,
+			Usage:    `The lowest log level that will be output. Accepted options are "debug", "info", "warn", "error"`,
+			Value:    "info",
+			EnvVars:  []string{common.PrefixEnvVar(envPrefix, "LOG_LEVEL")},
+		},
+		&cli.StringFlag{
+			Name:     common.PrefixFlag(flagPrefix, PathFlagName),
+			Category: category,
+			Usage:    "Path to file where logs will be written",
+			Value:    "",
+			EnvVars:  []string{common.PrefixEnvVar(envPrefix, "LOG_PATH")},
+		},
+		&cli.StringFlag{
+			Name:     common.PrefixFlag(flagPrefix, FormatFlagName),
+			Category: category,
+			Usage:    "The format of the log file. Accepted options are 'json' and 'text'",
+			Value:    "json",
```

Comment:
text?

Reply:
can update - but this is the default for eigenda

Reply:
Does it matter? Very unclear when we'll actually merge the two. And in the meantime we'll have to read these logs.

Reply:
Also this is a breaking change for users of this right? They might have had infra setup to ingest our previous log formats. Were those text?

Reply:
lemme evaluate - if both `op-log` and `eigensdk-go/logger` use the same slogger under the hood then it should be ok. I flagged this as a risk in the initial PR description.

Reply:
yeah output formats change: https://www.diffchecker.com/hFA97XKJ/ 

Let's just make sure to be very loud in our comms about this potential observability risk?

Reply:
Yeah its not that bad. still a golang format key-value pair output, so should be fine.
<img width="1225" alt="image" src="https://github.com/user-attachments/assets/e49a86e3-1431-43ac-a1b1-cff46b610cfd" />

Something looks off though. Some lines use different logging format than others.